presto-tpch/src/main/java/com/facebook/presto/tpch/TpchMetadata.java

```diff
@@ -487,8 +491,8 @@ public static Type getPrestoType(TpchColumn<?> column)
                 return INTEGER;
             case DATE:
                 return DATE;
-            case DOUBLE:
-                return DOUBLE;
+            case DECIMAL:
+                return DECIMAL;
```

Comment:
you should return DECIMAL(12,2), use com.facebook.presto.spi.type.DecimalType#createDecimalType(int, int)

Reply:
Ack